src/components/Footer/ClearButton/ClearButton.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import React, { useContext } from 'react';
+import { FooterContext } from '../../../context/FooterContext';
+
+export const ClearButton: React.FC = React.memo(() => {
```

Comment:
Add a type for the React.memo() function to ensure proper typing: React.memo<React.FC>(...).

Reply:
if I change this
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/87998640/c149857e-2134-4dcb-8f07-f97f89ca3e7c)
to this
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/87998640/e822aa80-e0b2-4006-8f54-1b560255ecb9)
I see mistake on ClearButton title
and if like this
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/87998640/d7375af2-ca4d-486e-b70c-2566176fe64d)
I see mistake here
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/87998640/db257a43-eadf-48f9-87ba-5db348225b33)
I'm confused 🤔